data/mil2525d/utilities/military-features-utilities/Bc2_and_C_CSV_generation_for_domains/Script/GenerateDomainCSV.py

```diff
@@ -92,7 +92,7 @@ def write_matching_rows_to_csv(CharlieName, the_query):
     choose_standard = arcpy.GetParameterAsText(2)  #Valid values are B2 or C
 
     if All_ID_Mapping_Original_csv == '':
-        All_ID_Mapping_Original_csv = '../Tooldata/All_ID_Mapping_Original.csv'
+        All_ID_Mapping_Original_csv = '../Tooldata/All_ID_Mapping_Latest.csv'
```

Comment:
Was there a reason for changing this?  

Also after the merge (or maybe you could also do as part of this PR) - I wanted to move these input files to the schema/domain inputs (the actual inputs to the tools rather than the tool internal data) as mentioned in https://github.com/Esri/military-features-data/pull/227#issuecomment-232100618